src/fragment-components/a-inline-loading.tsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import {
+	RadioButtonGroup,
+	RadioButton,
+	InlineLoading,
+	TextInput,
+	NumberInput
+} from 'carbon-components-react';
+import { AComponent } from './a-component';
+import { ComponentInfo } from '.';
+import { css } from 'emotion';
+import image from './../assets/component-icons/inline-loading.svg';
+import {
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj,
+	angularClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+
+export const AInlineLoadingSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<legend className='bx--label'>Status</legend>
+		<RadioButtonGroup
```

Comment:
Use dropdown instead with active as default.

Users will rarely need to change this.